docs/getting-started/installing/troubleshooting.md

```diff
@@ -0,0 +1,156 @@
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /app/app/etcd_client.rb:22:in `initialize'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /app/config.ru:11:in `new'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /app/config.ru:11:in `block in <main'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/rack-1.6.4/lib/rack/builder.rb:55:in `instance_eval'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/rack-1.6.4/lib/rack/builder.rb:55:in `initialize'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /app/config.ru:1:in `new'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /app/config.ru:1:in `<main'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/thin-1.7.0/lib/rack/adapter/loader.rb:33:in `eval'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/thin-1.7.0/lib/rack/adapter/loader.rb:33:in `load'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/thin-1.7.0/lib/rack/adapter/loader.rb:42:in `for'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/thin-1.7.0/lib/thin/controllers/controller.rb:170:in `load_adapter'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/thin-1.7.0/lib/thin/controllers/controller.rb:74:in `start'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/thin-1.7.0/lib/thin/runner.rb:200:in `run_command'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/thin-1.7.0/lib/thin/runner.rb:156:in `run!'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/lib/ruby/gems/2.3.0/gems/thin-1.7.0/bin/thin:6:in `<top (required)'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/bin/thin:23:in `load'
+ 2017-03-14T00:48:46.000Z kontena-ipam-plugin:   /usr/bin/thin:23:in `<top (required)'
+```
+
+
+## Possible causes
+
+### Kontena agent peer interface is incorrect
+
+Kontena nodes comunicates via a configured peer network which defaults to __eth1__ and fallback to __eth0__. 
+
+If your distribution or environment is using a different interface name, comunication is compromised.
+
+**Solution:** 
+
+Edit the file __/etc/kontena-agent.env__ and add `KONTENA_PEER_INTERFACE` variable to the correct interface. Repeat the same for __/etc/kontena-server.env__ file in the master nodes.
+
+Example:
+
+```
+KONTENA_PEER_INTERFACE=ens33
+```
+
+### Cloned docker services have the same docker id
+
+Kontena uses the docker id to identify nodes, if that is the case, things fail in un-expected ways. Also there are some components with some state stored in the server.
+
+You can check the docker id with the comand: `docker info | grep ID:`
+
+**Solution:**
+
+1. You could clone your server **before** installing docker and kontena.
+2. You could remove all traces of installation and do a clean install.   
```

Comment:
Of which installation? Whole Kontena enviroment is not needed to remove, just fix dockers on the nodes to have different ids. One option is to remove `/etc/docker/key.json`or `~/.docker/key.json` file and restart docker service.